samples/modules/lvgl/accelerometer_chart/README.rst

```diff
@@ -33,6 +33,12 @@ Requirements
 Building and Running
 ********************
 
+First, ensure the optional LVGL module is enabled and available:
+
+   .. code-block:: console
+      west config manifest.group-filter +optional
```

Comment:
Narrow down to a project filter (ref https://github.com/zephyrproject-rtos/west/issues/696)

```suggestion
      west config manifest.project-filter -- +lvgl
```